libs/ngxtension/create-notifier/src/create-notifier.ts

```diff
@@ -0,0 +1,12 @@
+import { signal } from '@angular/core';
+
+export function createNotifier() {
+	const sourceSignal = signal(0);
```

Comment:
Could think about something like, maybe:

```typescript
const sourceSignal = signal<'FIRST_EMISSION' | number>('FIRST_EMISSION'); // or 'INIT'?

// Then on the notify() updater
return {
  notify: () => {
	  sourceSignal.update((v) => v === 'FIRST_EMISSION' ? 0 : (v+1)); // narrowing ftw :D
  },
  listen: sourceSignal.asReadonly(),
}

// In the effect, if we want to skip the first emission, the code is more readable this way and tells exactly what happened
effect(() => {
  if(myTrigger.listen() === 'FIRST_EMISSION') {
    return;
  }
});

```

Reply:
I also prefer the less verbose option here but it's a closer call for me here, in this case `if(myTrigger.listen())` is a bit more mysterious. Again happy to wait and see what the general consensus is on this one, I don't mind it but I would probably prefer a slightly different approach. Maybe even having a separate computed signal e.g:

```ts
if(myTrigger.listenAfterInit()){
  // do stuff
}
```